interfaces/builtin/modem_manager.go

```diff
@@ -746,6 +746,9 @@ ACTION!="add|change|move", GOTO="mm_usb_device_blacklist_end"
 SUBSYSTEM!="usb", GOTO="mm_usb_device_blacklist_end"
 ENV{DEVTYPE}!="usb_device",  GOTO="mm_usb_device_blacklist_end"
 
+# Telegesis zigbee dongle
```

Comment:
Please add a complete name of the module in the comment. I take it's this one https://www.silabs.com/products/wireless/mesh-networking/telegesis-modules-gateways/etrx3-zigbee-usb-sticks ? The comment should be then 
```
# Telegesis ETRX3USB zigbee USB Stick
ATTRS{idVendor}...
```